libass/ass_types.h

```diff
@@ -28,6 +28,10 @@
 #define HALIGN_LEFT 1
 #define HALIGN_CENTER 2
 #define HALIGN_RIGHT 3
+#define JUSTIFY_AUTO 0
+#define JUSTIFY_LEFT 1
+#define JUSTIFY_CENTER 2
+#define JUSTIFY_RIGHT 3
```

Comment:
As a last change, could you prefix those with ASS_?

I know the other defines don't do this, but it's a good good convention to follow (since these are public symbols) and I'd argue the other defines should be switched to this convention at some point.